dashboard/app/models/script.rb

```diff
@@ -584,15 +589,18 @@ def self.setup(custom_files)
 
       # Stable sort by ID then add each script, ensuring scripts with no ID end up at the end
       added_scripts = scripts_to_add.sort_by.with_index {|args, idx| [args[0][:id] || Float::INFINITY, idx]}.map do |args|
-        add_script(*args)
+        add_script(args[0], args[1], new_suffix: new_suffix)
```

Comment:
I think this would work instead, and be a bit clearer:
```
...}.map do |options, raw_stages|
  add_script(options, raw_stages, new_suffix: new_suffix)
end
```